Sources/Commands/SwiftPackageTool.swift

```diff
@@ -68,11 +68,61 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
 
         case .update:
             let workspace = try getActiveWorkspace()
-            try workspace.updateDependencies(
-                root: getWorkspaceRoot(),
-                diagnostics: diagnostics
-            )
+            if options.packageUpdateDryRun {
```

Comment:
We should add one test for this feature. It can go in the PackageToolsTests.swift file.